src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow weak
```

Comment:
Would it be possible to add any type annotations if you're already using flow? I know there have been some recent efforts by @vjeux to add annotations. I don't see any files that are using weak mode either so that might be something to consider.